sdk/clientcore/core/src/test/java/io/clientcore/core/shared/HttpClientTests.java

```diff
@@ -79,6 +79,7 @@
 import static io.clientcore.core.http.models.ResponseBodyMode.DESERIALIZE;
 import static io.clientcore.core.http.models.ResponseBodyMode.IGNORE;
 import static io.clientcore.core.http.models.ResponseBodyMode.STREAM;
+import static io.clientcore.core.implementation.util.ImplUtils.bomAwareToString;
```

Comment:
In clientcore, both CoreUtils and ImplUtils are in implementation. We should merge those into a single utils class.